### DISCUSSION TITLE
Will you be supporting multi-canvas re-ordering?

### DISCUSSION THREAD

**Discussion by brittnylapierre** [digirati-co-uk/iiif-manifest-editor#127] [Q&A] [answered]
Hello! I am wondering if you will be supporting re-arranging multiple canvases at once in your manifest editor? Thanks

**Comment by tomcrane** [comment, marked as answer]
Hi Brittny

We were going to implement this in two stages:
https://github.com/digirati-co-uk/iiif-manifest-editor/issues/71

Initially, just moving one canvas at a time.
We will come back and support multiple canvas moving later on.

Is this an important use case for you? Do you do this often and what kinds of material are you editing?

**Reply by brittnylapierre** [reply]
Thank you!

I work for CRKN, the non-profit that runs https://www.canadiana.ca/ We're an organization dedicated to the preservation and access of Canada’s documentary heritage. 

We have transitioned our data model to IIIF and are looking for open-source software that we can leverage and contribute to. We want to be able to participate in the community as well as offer better services to our members by doing so.

We have many manifests which represent scanned documents, maps,  books, etc. We have some collections that make it easier to navigate the archive from our access platform, canadiana.ca.

We are looking for a manifest editor that will allow our platform admins to manage the content and metadata of our manifests. We have a simple admin interface that I have developed with a bare-bones/MVP manifest editor that will be replaced with the open-source solution that best aligns with our admin team's needs. 

Our admin team has highlighted a use case where they would need to merge manifests. (Either duplicate versions of the same scanned object, or two separate objects.) Sometimes it would be a complete merge of the two manifests, while other times it would be taking some pages from one of the manifests and putting them into the other manifest. 

Their wish list item would be to easily select multiple canvases and rearrange them within the editor. They would love this to be a drag and drop feature.

They said this would be a high-priority feature. In the manifests that they've had to work on, they often have batches of canvases near each other that need to be rearranged.

I would love to arrange a meeting between your team and my coworkers if you would like to gather more information from them!

We're excited about this editor and look forward to getting to try it out and see its development progress. If you ever require more contributors I would love to help out, too.
